src/components/timeseries-chart/timeseries-chart.tsx

```diff
@@ -277,7 +277,8 @@ export const TimeseriesChart: React.FC<TimeseriesChartProps> = ({
           onFetchData={onFetchData}
           pointsPerSeries={pointsPerSeries}
           collections={collectionToRender}
-          timeDomain={[+start, +end]}
+          timeDomain={timeDomain?.map((td) => +td)}
+          timeSubDomain={timeSubDomain?.map((tsd) => +tsd)}
```

Comment:
what happens if timeSubdomain is undefined? Can we instead do

`timeSubDomain ? timeSubdomain.map((tsd) => +tsd)} : null` ?

Reply:
This is normal behaviour. Undefined and null is both falsy values. Short answer, nothing will happen

Reply:
if you try to map through undefined though, you will get an error:

<img width="388" alt="Screenshot 2021-01-21 at 08 03 43" src="https://user-images.githubusercontent.com/66323467/105315244-50f44900-5bbf-11eb-8d3f-d063f8db2e18.png">

Reply:
dont you see the ? mark? it will not run map if timeSubDomain is undefined

Reply:
https://www.typescriptlang.org/play?#code/MYewdgzgLgBAhmAggJ2XAnjAvDArmAEwFMAzASzCIPgjzAGswQB3MGmaZCgcwG0BdANwAoUJBAAbIgDoJIbgAoABghRp0ALhgASAN6rUGAL5KAlMIPqA-NIC2cAA4KAbtgB8MMREky5i56amIl4+svIKAEQAso4OPHTE5JTUcIaYBGQEYLAkcGQSEUHCMDBAA

Reply:
the proof is in the pudding
![Screenshot 2021-01-21 at 08 32 37](https://user-images.githubusercontent.com/404102/105317977-3a4ff100-5bc3-11eb-82eb-9260bd40c2a6.png)

Reply:
ah! okay I understand, thanks. 👍